build.gradle.kts

```diff
@@ -165,15 +168,11 @@ testing {
         val journeyTest by registering(JvmTestSuite::class) {
             useJUnitJupiter()
             dependencies {
-                implementation(project())
-                implementation("org.springframework.boot:spring-boot-starter-test") {
+                implementation(libs.org.springframework.boot.spring.boot.starter.test) {
                     exclude("org.mockito", "mockito-core")
                     because("Use MockK instead of Mockito since it is better suited for Kotlin")
                 }
-                implementation("org.springframework.boot:spring-boot-starter-webflux")
-                implementation("org.springframework.security:spring-security-test")
-                implementation("org.awaitility:awaitility:4.2.0")
-                implementation("org.jsoup:jsoup:1.16.1")
```

Comment:
Same here as in the unit test dependencies I assume

Reply:
Correct.